packages/tools/fuzzer/ipc/Client.ts

```
@@ -1,53 +1,69 @@
+import ProgressBar from '../utils/ProgressBar';
+import logDiffDebug from '../utils/logDiffDebug';
 import { NoteEntity } from '@joplin/lib/services/database/types';
-import diffSortedStringArrays from './utils/diffSortedStringArrays';
-import extractResourceIds from './utils/extractResourceIds';
+import diffSortedStringArrays from '../utils/diffSortedStringArrays';
+import extractResourceIds from '../utils/extractResourceIds';
 import { substrWithEllipsis } from '@joplin/lib/string-utils';
-import hangingIndent from './utils/hangingIndent';
+import hangingIndent from '../utils/hangingIndent';
+import { readFile, writeFile } from 'fs/promises';
+import { hasOwnProperty } from '@joplin/utils/object';
```

Biome (2.3.14)

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)